ts/features/idpay/details/screens/IdPayInitiativeDetailsScreen.tsx

```diff
@@ -97,11 +102,41 @@ const InitiativeDetailsScreen = () => {
     initiativeNeedsConfigurationSelector
   );
 
+  if (!pot.isSome(initiativeDataPot)) {
```

Comment:
why not use `pot.isLoading` here? wouldn't this trigger if there was an error fetching the initiative aswell?

Reply:
Exactly! We currently show the skeleton component for error and loading cases.